jwe/jwe_test.go

```diff
@@ -0,0 +1,56 @@
+package jwe_test
```

Comment:
It would be nice if we could re-use example data from https://datatracker.ietf.org/doc/html/rfc7516#appendix-A